### ISSUE TITLE
Assertion failures with some perl versions

### ISSUE THREAD

**Issue by eserte** [dallaylaen/namespace-local-perl#3]
t/10-isa.t fails with some perl versions, possibly only if the perl is compiled with `-DDEBUGGING`. With a perl5.18.1:
```
perl5.18.1: mg.c:1629: Perl_magic_clearisa: Assertion `((((_gvstash)->sv_flags & (0x00004000|0x00008000)) == 0x00008000) && (((svtype)((_gvstash)->sv_flags & 0xff)) == SVt_PVGV || ((svtype)((_gvstash)->sv_flags & 0xff)) == SVt_PVLV))' failed.
t/10-isa.t ................ 
No subtests run 
```
Here's a similar fail report at CPAN Testers:
http://www.cpantesters.org/cpan/report/23a2e2a6-00c5-11e9-b99f-8628edda83cf

**Comment by dallaylaen** [comment]
Yes, thanks. It seems 0.0604 fixes that. Let's see how it goes...

**Comment by dallaylaen** [comment]
Finally fixed in v.0.07_07 by namespace-local-perl@9fe09c2

**Comment by dallaylaen** [comment]
Fixed in 9fe09c2abaeb8976e512b47ac22c554922ab6a27
